From 79222529e56ae395d6e9b69b326262f3bde453e1 Mon Sep 17 00:00:00 2001
From: Bai Ping <b51503@freescale.com>
Date: Fri, 11 Dec 2015 14:59:10 +0800
Subject: [PATCH 0768/5242] MLK-11996 ARM: imx: fix ddr_type config error

commit  78bf6bf93e193020cd74e03d74f3ade15e547646 from
https://source.codeaurora.org/external/imx/linux-imx.git

The config option for getting DDR type should
be 'HAVE_IMX_MMDC' and 'HAVE_IMX_DDRC'. Otherwise,
get_ddr type will always return 0(DDR3).

Signed-off-by: Bai Ping <b51503@freescale.com>
(cherry picked from commit 6e8048311f854184ae5f16c822e6f6f0fd122e54)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/mach-imx/common.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-imx/common.h b/arch/arm/mach-imx/common.h
index 0476e63..ae8bbef 100644
--- a/arch/arm/mach-imx/common.h
+++ b/arch/arm/mach-imx/common.h
@@ -138,7 +138,7 @@ static inline void imx_gpcv2_set_core1_pdn_pup_by_software(bool pdn) {}
 #else
 static inline int imx_mmdc_get_ddr_type(void) { return 0; }
 #endif
-#ifdef CONFIG_HAVE_DDRC
+#ifdef CONFIG_HAVE_IMX_DDRC
 int imx_ddrc_get_ddr_type(void);
 #else
 static inline int imx_ddrc_get_ddr_type(void) { return 0; }
-- 
1.7.9.5

